Doc/library/asyncio-sync.rst

```diff
@@ -66,6 +66,13 @@ Lock
       This method waits until the lock is *unlocked*, sets it to
       *locked* and returns ``True``.
 
+      When more than one coroutine is blocked in :meth:`acquire`
+      waiting for the lock to be unlocked, only one coroutine
+      eventually proceeds.
+
+      Acquiring a lock is *fair*: the coroutine that proceeds will be
```

Comment:
I don't quite understand what you mean by "fair"? It reads oddly to me.

Reply:
"Fair" is a standard term for the kind of lock that avoids starvation; see e.g. [here](https://www.threadingbuildingblocks.org/docs/help/tbb_userguide/Mutex_Flavors.html) by treating waiters as first-come first-serve.

It should be fine if someone is not familiar with the term because the rest of the sentence explains it.

Reply:
I see *fair* in multithreading context first time here.
Not sure if this is *standard term*.
Can we avoid this word in documentation?

Reply:
The term _fair_ is widely used in this meaning across programming languages, e.g. [kotlin](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.sync/-mutex/lock.html), [java](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/locks/ReentrantLock.html), [golang](https://golang.org/src/sync/mutex.go), [c++](https://stackoverflow.com/questions/17527621/is-stdmutex-fair), [Python](https://stackoverflow.com/questions/8158442/fair-semaphore-in-python).

The idea is to use the technical term to help people already familiar with the concept, and also explain it in the rest of the sentence. That kind of usage doesn't hinder understanding, and it can even help the reader when encountering the term later.

Reply:
In mentioned links *fairness* is used in java. kotlin inherits this term. 
Golang has fairness in documentation only.
C++ and Python links point on stackoverflow which is nice site but not the official python documentation.

Docs for [python threading](https://docs.python.org/3/library/threading.html) doesn't mention fairness.  I think if we add the term we should enumerate it in [python glossary](https://docs.python.org/3/glossary.html) as well.

I still not convinced that this is required

Reply:
@1st1 what is your opinion?

Reply:
I think using "fair" is fine here, but not really necessary as the text after the colon explains it in detail anyways.

Reply:
Ok, if Yuri agree